pkg/resources/config.go

```diff
@@ -21,6 +21,21 @@ var replicaSizes = []string{
 	"4xlarge",
 	"5xlarge",
 	"6xlarge",
+	"25cc",
```

Comment:
I noticed you replaced 3xsmall with 50cc, is 25cc even smaller? Or is 50cc just a better recommendation for production use?

Reply:
It is mainly for the tests, the actual cluster sizes on the dev Materialize container are actually configured to be the same:

https://github.com/MaterializeInc/terraform-provider-materialize/blob/29233a0abd848e733e9dc307efd57fa63c1d1b7d/docker-compose.yml#L10

But this is a good point, I've switched the tests to 25cc for consistency.